common/internal-types/Cargo.toml

```diff
@@ -12,10 +12,16 @@ license = "GPL-3.0-only"
 crate-type = ["rlib"]
 
 [dependencies]
+alloy = { workspace = true, default-features = false, features = [
```

Comment:
Does this crate need to depend on `alloy` and if so, are all the enabled features here required?